openedx/core/djangoapps/content/learning_sequences/api/outlines.py

```diff
@@ -80,11 +80,19 @@ def key_supports_outlines(opaque_key: OpaqueKey) -> bool:
     return False
 
 
+@function_trace('learning_sequences.api.get_course_keys_with_outlines')
```

Comment:
Is default name of function name not good enough? Does it provide any other information? It's too bad that we need to copy the function name with potential copy/paste issues.  Just curious.